src/EtherFiNodesManager.sol

```diff
@@ -95,6 +103,76 @@ contract EtherFiNodesManager is
         IEtherFiNode(etherfiNodeAddress(id)).startCheckpoint();
     }
 
+    /**
+     * @notice Forwards EIP-7002 withdrawal requests to the EigenPod, supporting single or batch requests.
+     * @dev Access: only addresses with ETHERFI_NODES_MANAGER_EL_TRIGGER_EXIT_ROLE.
+     * @dev Pausable + nonReentrant for safety.
+     * @param pod The EigenPod address that owns the validators.
+     * @param requests An array of WithdrawalRequest, where:
+     *        - requests[i].pubkey is the 48-byte BLS pubkey
+     *        - requests[i].amountGwei == 0 means "full exit"; >0 means partial to pod
+     * @custom:fee You MUST send sufficient ETH in msg.value to cover the EIP-7002 predeploy fee
+     *             for ALL requests in this batch (fee updates per block). Overpay is okay.
+     */
+    function batchWithdrawalRequests(
+        address pod,
+        IEigenPod.WithdrawalRequest[] calldata requests
+    ) external payable whenNotPaused nonReentrant
+    {
+        // ---------- checks ----------
+        if (pod == address(0)) revert UnknownNode();
+        if (!roleRegistry.hasRole(ETHERFI_NODES_MANAGER_EL_TRIGGER_EXIT_ROLE, msg.sender)) revert IncorrectRole();
+        uint256 n = requests.length;
+        if (n == 0) revert EmptyWithdrawalsRequest();
+
+        // Rate-limit by number of validator requests
+        uint64 units = SafeCast.toUint64(n);
+        bool ok = BucketLimiter.consume(exitRequestsLimit, units);
+        if (!ok) revert ExitRateLimitExceeded();
+
+        // Strict pubkey length check to avoid wasting fee on malformed requests.
+        for (uint256 i = 0; i < n; ) {
+            bytes memory pubkey = requests[i].pubkey;
+            // Compute the pubkey hash
+            bytes32 pubkeyHash = keccak256(pubkey);
+
+            // Ensure the node exists for this pubkey hash
+            if (address(etherFiNodeFromPubkeyHash[pubkeyHash]) == address(0)) {
```

Comment:
this is wrong. Pubkey hash is not via keccak, it is via the SSZ serialization spec to match how eigenlayer computes it. you can use `calculateValidatorPubkeyHash`